.clang-format

```diff
@@ -0,0 +1,28 @@
+BasedOnStyle: LLVM
+IndentWidth: 8
+UseTab: ForIndentation
+BreakBeforeBraces: Attach
```

Comment:
Most of the code is really `Attach`, but I got used to `Linux` here, which I find more readable (opening brace of function starts on new line) and I already used it some of the code already. This is something probably not for a change, but maybe for consideration.

Reply:
I think this would be changing the `AfterFunction` value before.